scripts/main.js

```diff
@@ -0,0 +1,15 @@
+//import services()
+//import locations()
+//import guests()
+
+//get container from index.html and save to a variable using document.query selector
+
+//set a base html string to be added to the dom later ie mainHTML
+
+//<h1>Cider Falls</h1> //logo here? 
+//<section>services()</section>
```

Comment:
Thinking forward, could use some id's and classes in each section for formatting purposes later.